src/api/app/controllers/webui/cloud/upload_jobs_controller.rb

```diff
@@ -46,10 +43,6 @@ def destroy
 
       private
 
-      def set_breadcrump
```

Comment:
This typo will be sorely missed :rofl:

Reply:
![3aa5wn](https://user-images.githubusercontent.com/514785/64715962-bd3a4500-d4c1-11e9-90a0-ec3018caedc6.jpg)